addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -206,13 +191,21 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
         }
     }
     .o_pricelist_dropdown, .o_sortby_dropdown {
-        min-width: fit-content;
+        //truncates the dropdown pricelist > 8rem
```

Comment:
I don't understand why is needed... watchout that once switched from "Benelux" to "EUR" my dropdown disappeared